is.js

```diff
@@ -1,4 +1,6 @@
 module.exports = {
   array: Array.isArray,
-  primitive: function(s) { return typeof s === 'string' || typeof s === 'number'; },
+  primitive: function(s) {
+      return typeof s === 'string' || typeof s === 'number' || typeof s === 'boolean';
```

Comment:
I've tried to change as little as possible. 
Right now this method is used to answer the question "could we use this value as a content of a text node?" and for `null` or `undefined` implicit answer is no.

More precisely, [here](https://github.com/valentin-nemcev/snabbdom/blob/d338319cf6e409a1c318228c62d9943d1b73f931/snabbdom.js#L78) we don't expect `null` or `undefined`, [here](https://github.com/valentin-nemcev/snabbdom/blob/d338319cf6e409a1c318228c62d9943d1b73f931/h.js#L16) we check for them before, and [here](https://github.com/valentin-nemcev/snabbdom/blob/d338319cf6e409a1c318228c62d9943d1b73f931/h.js#L16) considering `null` and `undefined` as primitives will change edge case behaviour, where if you have these values in a list of children, snabbdom will insert text nodes for them instead of throwing exception.

I also want to make another PR to skip `null` and `undefined` values in a list of children, for same reasons as this PR. Then you could consider them primitive, so the method behaviour would be more aligned with its name.

I just didn't considered symbols, probably they could be included in the list.